requirements.txt

```diff
@@ -1,12 +1,5 @@
 # START
-
-# direct requirements
--e git+git://github.com/inveniosoftware/invenio-jsonschemas.git#egg=invenio-jsonschemas
-
-# requirements to unbreak current DEV situation
--e git+git://github.com/inveniosoftware/invenio-oauth2server.git#egg=invenio-oauth2server
--e git+git://github.com/inveniosoftware/invenio-previewer.git#egg=invenio-previewer
--e git+git://github.com/inveniosoftware/invenio-webhooks.git#egg=invenio-webhooks
+-e git+git://github.com/inveniosoftware/invenio-deposit.git#egg=invenio-deposit
 
```

Comment:
Good, all dependencies are on PyPI except `deposit` that we should also release.